lib/span-data.js

```diff
@@ -28,10 +28,11 @@ var uid = 1;
  * @param {string} name The name of the span.
  * @param {number} parentSpanId The id of the parent span, 0 for root spans.
  * @param {boolean} isRoot Whether this is a root span.
+ * @param {number} skipFrames the number of frames to remove from the top of the stack.
  * @param {number=} time The time when the associated TraceSpan begins.
```

Comment:
Aside: do we ever call this constructor with the `time` argument in the codebase? Perhaps it should be dropped (in a follow-on)